x-pack/test/plugin_functional/plugins/audit_trail_test/kibana.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "audit_trail_test",
```

Comment:
TODO: decide on AuditTrail interface and move this test under integration tests with jest